trino-connector/src/main/java/com/datastrato/gravitino/trino/connector/catalog/CatalogConnectorManager.java

```diff
@@ -56,7 +67,7 @@ public class CatalogConnectorManager {
 
   private GravitinoClient gravitinoClient;
   private GravitinoConfig config;
-  private String usedMetalake;
+  private final Set<String> usedMetalakes = new HashSet<>();
```

Comment:
Is this change still required?

Reply:
We need to support configuring multiple connectors, each using a different metastore.